content/docs/04.5-devx/05-app-profile.md

```diff
@@ -93,4 +96,73 @@ spec:
 
 </WarningBox>
 
+# App Profile Macros
+
+Pack macros are the variables defined in the App Profile and are only resolved at the cluster deployment time. The app profile variables can be:
+
+* directly given by the developers while modelling the application profiles.
+
+* resolved during the app instatiation.
+
+* funnelled/inherited from the lower tiers of the profile.
+
+The variables are of two types:
+
+* [The Input Parameters](/devx/app-profile#inputparameters)
+
+* [The Output Parameters](/devx/app-profile#outputparameters)
+<br />
+
+## Input parameters
+
+What user can specify and will be consumed by the current tier. It can even use output variables from below tier. This parameter resolves to a value of the variable defined in App Profile tier parameters.yaml file.
+
+```
+{{.spectro.system.[VARIABLE_NAME]}}
+```
+|**Supported Variables**|  
+|-----------------------|
+|appprofile.name| 
+|appprofile.uid| 
+|appdeployment.name| 
+|appdeployment.uid|
+|appdeployment.tiername: Resolves to a string value in the format `<deployment name>-<tier name>`.|
```

Comment:
Make this a two columns table with the description in the other columns. Also, if one row has a description, then all other rows should also have a description.